### DISCUSSION TITLE
How to synchronize notes within a folder in WebDav?

### DISCUSSION THREAD

**Discussion by Kmiyh** [remotely-save/remotely-save#430] [Q&A] [answered]
How to synchronize notes within a folder in WebDav? There is a storage root, I create a folder in it from my phone and in the folder there are several notes. When I synchronize the folder and the files in it get into WebDav storage, but if I try to log in from my computer, I can only see the folders themselves, but no notes in them

**Comment by fyears** [comment, marked as answer]
please try to set "depth head=1" in the setting page of remotely-save and try again.

and may i ask what webdav service are you using?

**Reply by Kmiyh** [reply]
Thank you! That solved my problem. I use Yandex (https://webdav.yandex.ru) as my WebDav.
